unreleased.md

```diff
@@ -4,3 +4,7 @@
 
 - `ObjectUserType` was causing its data to be corrupted when it contains special
   characters, fixed.
+
+## Improvements
+
+- `GenericADOExceptionHandler` added and server response improved.
```

Comment:
please don't use implementation details in release notes. just say what is improved.